serverless.yaml

```diff
@@ -10,18 +10,20 @@ custom:
   resourceTableName: 'resource-db-${self:custom.stage}'
   stage: ${opt:stage, self:provider.stage}
   region: ${opt:region, self:provider.region}
+  kibanaUserEmail: ${opt:kibanaUserEmail, 'abc@xyz.com'}
```

Comment:
I don't think we want a default Kibana user email address

Reply:
Will replace it with an empty string